pentasharp/Program.cs

```diff
@@ -9,6 +12,9 @@ public static void Main(string[] args)
             // Add services to the container.
```

Comment:
remove all comments in this file please, consider it as a clean-up of the class that was not created by you initially